htdocs/web_portal/components/Draw_Components/menu.php

```diff
@@ -100,10 +100,10 @@ function add_menu_item($menu_item)
     }
     if ($show == "all" or ($show=="write_enabled" and (!$portalIsReadOnly or $userisadmin)) or (($show == "admin") and ($userisadmin))){
         $html .=    	"<li class=\"Menu_Item\">".
-                        "<a href=\"".$link."\"><span class=\"menu_link\">".
+                        "<a href=\"".htmlspecialchars($link)."\"><span class=\"menu_link\">".
```

Comment:
How's this work?

Reply:
http://php.net/manual/en/function.htmlspecialchars.php

Reply:
Ah, ok. I wasn't sure if it was a PHP thing or your own function that was somewhere else.